packages/react-static/src/static/plugins.js

```diff
@@ -1,39 +1,6 @@
 import { getHooks, reduceHooks } from '../utils'
 
-const supportedHooks = [
```

Comment:
What's the reason these ae removed? Even though they don't have a default implementation, are you certain they're not actually supported anymore?

Reply:
All the entries in the array are also keys in the exported object. So I just removed the duplication. validatePlugin now takes the valid hook names from the exported object.

Reply:
GitHub was collapsing those lines, so I didn't realise. This is good. Thanks :)